components/expanding-components/section.scss

```diff
@@ -0,0 +1,52 @@
+.msds-expanding-section {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  // padding-top: 96px;
+  // padding-bottom: 80px;
+
+  &--closed {
+    display: flex;
```

Comment:
i dont think you need display flex here as its already set on the outer selector. You only need to specify if you want to overwrite something

Reply:
I need to set it to flex, as I toggle it to display none/flex when the user clicks on the floating button